plugins/Live/lang/en.json

```diff
@@ -1,69 +1,69 @@
 {
     "Live": {
-        "AveragePageGenerationTime": "Each page took on average %1$s to load for this visitor.",
+        "AveragePageGenerationTime": "Each page took %1$s to load for this visitor on average.",
         "CalculatedOverNPageViews": "Calculated using this visitor's last %1$s page views.",
-        "ClickToViewMoreAboutVisit": "Click to view more information about this visit",
-        "ClickToViewAllActions": "Click to view all actions of this group in detail",
+        "ClickToViewMoreAboutVisit": "Click for more info about this visit",
+        "ClickToViewAllActions": "Click for all actions of this group in detail",
         "ConvertedNGoals": "Converted %s Goals",
         "FirstVisit": "First visit",
         "GoalType": "Type",
         "HideMap": "hide map",
-        "KeywordRankedOnSearchResultForThisVisitor": "The keyword %1$s was ranked %2$s on the %3$s search result page for this visitor",
+        "KeywordRankedOnSearchResultForThisVisitor": "The keyword \"%1$s\" was ranked %2$s on the %3$s search result page for this visitor",
         "LastHours": "Last %s hours",
         "LastMinutes": "Last %s minutes",
         "LastVisit": "Last visit",
-        "LinkVisitorLog": "View detailed visits log",
-        "LoadMoreVisits": "Load more visits",
-        "LimitedSummary": "The information in this profile summarizes the last %1$s visits. This user had more visits in total.",
+        "LinkVisitorLog": "View detailed visit log",
```

Comment:
"Log of visits" awkward but passable, or "Visit log" works.
Same reason it isn't the "Users guide" or "Spices rack".
I find it corny to present generic things as proper nouns, when it could be "settings", bookmarks" and so on.

Reply:
Thanks for the explanation. As mentioned before, I'm not a native English speaker.
If it's better English to write `visit log` instead of `visits log`, I'm fine with changing that.

@tsteur @mattab Is it fine if we change the name of that feature? As we are using the name in some other places (like on the website), we maybe should consider changing it there as well at some point 🤔

Reply:
I'd prefer to stick to `visits log` for now since that's the name of the feature for now, and AFAIK it's still OK in english to write it that way (although maybe the other way is more "natural" it's not needed to change and having to rename it everywhere etc.)

Reply:
There are 20 instances of "Visits log" (outside of website use).
12 of them are in this plugin, whereas the rest are in
_Corehome_
_GoogleAnalyticsImporter_ and
_Tour_
which will get picked up as I get to it.
The idea that people are going to find a discrepancy coming from the website or between plugins is, dubious.
More likely a compounding effect of reading stuff that is off and the amount of places it happens is mitigated.

Reply:
Thanks for that, but I prefer to keep "Visits log"